src/esp/metadata/attributes/PhysicsManagerAttributes.cpp

```diff
@@ -10,9 +10,10 @@ namespace attributes {
 
 PhysicsManagerAttributes::PhysicsManagerAttributes(const std::string& handle)
     : AbstractAttributes("PhysicsManagerAttributes", handle) {
-  setSimulator("none");
-  setTimestep(0.01);
-  setMaxSubsteps(10);
+  setSimulator("bullet");
```

Comment:
I copied these values from our `default.physics_config.json` in `master` (minus the `rigid object paths` entry, which appears to be obselete).